Lib/test/test_type_aliases.py

```diff
@@ -317,6 +317,17 @@ def test_module(self):
         self.assertEqual(mod_generics_cache.OldStyle.__module__,
                          mod_generics_cache.__name__)
 
+    def test_unpack(self):
+        type Alias = tuple[int, int]
+        unpacked = (*Alias,)[0]
+        self.assertEqual(unpacked, Unpack[Alias])
+
+        class Foo[*Ts]:
+            pass
+
+        x = Foo[str, *Alias]
+        self.assertEqual(x.__args__, (str, Unpack[Alias]))
```

Comment:
You probably want `get_args` and not `__args__` here.

Reply:
I actually don't need any of the extra magic done by `get_args`, so I think `__args__` might be better suited here?